src/Orm/Zed/StoreContext/Persistence/SpyStoreContext.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
I'm not sure that this is related to the current task

Reply:
Yes but we have to update modules with features and this classes came with new feature's module, the CI is red without it

Reply:
I see, ok